app/src/processing/app/Editor.java

```diff
@@ -1374,6 +1374,24 @@ public void actionPerformed(ActionEvent e) {
 
     menu.addSeparator();
 
+    JMenuItem increaseFontSizeItem = newJMenuItem(tr("Increase Font Size"), '=');
```

Comment:
Could work, I'm ccing our UX designer for an advice :slightly_smiling_face:  @00alis